core/thread/inc/ThreadPool.h

```diff
@@ -48,7 +51,7 @@ class ThreadPool: public TPool<ThreadPool> {
    using TPool<ThreadPool>::Reduce;
 
 private:
-   tbb::task_scheduler_init *initTBB = new tbb::task_scheduler_init(tbb::task_scheduler_init::deferred);
+   tbb::task_scheduler_init *fInitTBB = new tbb::task_scheduler_init(tbb::task_scheduler_init::deferred);
```

Comment:
Could you replace this with a `unique_ptr`? A plain `new` in a in-class member initializer is both uncommon (thus surprising) and a dangerous source of leaks...